kit/themes/src/UILabel/index.tsx

```diff
@@ -4,3 +4,6 @@ import { ColorVariants } from '../Colors';
 export * from './UILabel';
 export const UILabelRoles = TypographyVariants;
 export const UILabelColors = ColorVariants;
+
+import type { Props } from './types';
```

Comment:
What do you think to rather rename `Props` to `UILabelProps` in place, to have proper `Cmd+Click` behaviour?

Reply:
Good point, I'll change it :)